courses/free-courses-hi.md

```diff
@@ -432,6 +432,7 @@
 
 #### React
 
+* [Learn React JS in Hindi](https://www.youtube.com/watch?v=6l8RWV8D-Yo&list=PL2PkZdv6p7ZnS2QIz8WAPmqeqUUccPNQY&index=3&t=17419s) - Coder Dost
```

Comment:
Don't invent titles! Use the title given by the resource.

Reply:
Applied the changes